padevents/padevents.py

```diff
@@ -166,6 +166,8 @@ async def check_started(self):
                             or event.server != aed['server'] \
                             or (aed['key'], event.key) in self.rolepinged_events:
                         continue
+                    if not aed['include3p'] and bool(re.match("^Multiplayer", event.clean_dungeon_name)):
```

Comment:
it's not actually necessary to use a regex here - we can just use `.startswith()`, and generally it's preferred to avoid regex when there's a not-regex option available, because regex can be kinda hard to read. `.startswith()` will return a bool already so the logic will be cleaner too, yay